source/manuals/programming-languages/ruby.html.md.erb

```diff
@@ -1,113 +1,50 @@
-
-[ruby-toolbox]: https://www.ruby-toolbox.com/categories/testing_frameworks
-[gorails]: https://gorails.com/tool_categories/test-frameworks/tools
-[chris-zetter-blogpost]: https://about.futurelearn.com/blog/how-we-write-readable-feature-tests-with-rspec
-
-### How to configure RSpec
-
-We should try to avoid the pollution of the global namespace that came with earlier versions of RSpec. For example, we should be using `RSpec.describe` instead of `describe`.
-
-This can be enforced by setting `config.expose_dsl_globally = false` in your test setup. Read [Global namespace DSL][Global namespace DSL] for more information.
-
-[Global namespace DSL]: https://relishapp.com/rspec/rspec-core/docs/configuration/global-namespace-dsl
-
-### Example unit test
-
-This example is taken from [finder-frontend][finder-frontend]:
-
-[finder-frontend]: https://github.com/alphagov/finder-frontend
-
-```ruby
-RSpec.describe ContentItem do
-  subject { described_class.new(base_path) }
-  let(:base_path) { "/search/news-and-communications" }
-  let(:finder_content_item) { news_and_communications }
-  let(:news_and_communications) {
-    JSON.parse(File.read(Rails.root.join("features", "fixtures", "news_and_communications.json")))
-  }
-
-  RSpec.describe "as_hash" do
-    it "returns a content item as a hash" do
-      expect(subject.as_hash).to eql(finder_content_item)
-    end
-  end
-end
-```
-
-### Example integration test
-
-This example is taken from [publishing-e2e-tests][publishing-e2e-tests]:
-
-[publishing-e2e-tests]: https://github.com/alphagov/publishing-e2e-tests
-
-```ruby
-# Some methods and variables have been removed here for brevity, but you get the idea.
-
-RSpec.feature "Publishing a parent and child topic on Collections Publisher" do
-  let(:parent_title) { unique_title }
-  let(:child_title) { unique_title }
-
-  RSpec.scenario "Publishing a parent and child topic" do
-    given_i_have_a_published_topic
-    when_i_add_a_child_topic
-    and_i_publish_it
-    then_i_can_view_both_on_gov_uk
-  end
-
-private
-
-  def signin_to_signon
-    signin_with_next_user("Collections Publisher" => ["GDS Editor"])
-  end
-
-  def then_i_can_view_both_on_gov_uk
-    click_link(link)
-    expect_url_matches_live_gov_uk
-    expect(page).to have_content(child_title)
-
-    first(:link, parent_title).click
-    expect_url_matches_live_gov_uk
-    expect(current_url).to end_with(parent_slug)
-  end
-end
-```
-
-[RuboCop]: https://github.com/rubocop-hq/rubocop 
-[rubocop-govuk]: https://github.com/alphagov/rubocop-govuk
\ No newline at end of file
+## Code formatting
+
+The GOV.UK programme publishes and maintains a gem, [rubocop-govuk], which
+provides linting rules for Ruby projects. These rules extend those
+described in [rubystyle.guide], which is a good starting point for
+the underlying conventions. The rubocop-govuk project has overriden rules to
+rules to either match pre-existing GDS prcedence or to handle scenarios where
```

Comment:
`rules to rules to`

Remove one of the `rules to`

Reply:
Thanks Dean 👍  By the way I've spotted various errors since opening (the curse of a "I did this at the end of a Friday") so there'll be a few fixes made in the next few mins, just in case you're actively reviewing.